intel-retail/automated-self-checkout
# Grpc go in pipeline profile

## PR Checklist



## What are you changing?

Add GRPc go pipeline profile.

## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- if haven't built the model-server yet, the first time you can build it with make build-ovms-server and it will take a while.
- build grpcgo image 
- cd configs/opencv-ovms/grpc_go && make build
- run camera simulator: ./camera-simulator/camera-simulator.sh
- build ovms-client: make build-ovms-client
- docker-run with PIPELINE_PROFILE env set to "grpc_go": PIPELINE_PROFILE="grpc_go" sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
- exam the ovms-client docker container log and should see no error: docker logs ovms-client0
- exam the result outputs and should see the pipeline fps output: sudo cat ./results/r0.jsonl

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
